MobileWallet/Screens/Send/SendingTari/SendingTariViewController.swift

```diff
@@ -320,7 +322,9 @@ class SendingTariViewController: UIViewController {
                     TariLogger.error(self.debugMessage)
                     //No success animation, just go home and display an error
                     self.removeTitleAnimation()
-
+                    if OnionManager.shared.state == .stopped {
```

Comment:
We're not stopping tor, it just needs to check if it hasn't connected yet

```suggestion
                    if OnionManager.shared.state != .connected {
```